hello-spring/src/test/java/org/folio/hello/tests/RollTest.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.hello.tests;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.both;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.lessThan;
+import static org.hamcrest.Matchers.everyItem;
+
+
+import java.util.Random;
+import java.util.Arrays;
+
+import io.restassured.response.Response;
+
+import org.folio.hello.BaseIntegrationTest;
+import org.junit.jupiter.api.Test;
+
+public class RollTest extends BaseIntegrationTest {
```

Comment:
Regular classes (and methods) don't need `public` (or any access modifiers); sonarlint should have warned about this.

The only exception is for when test classes are extended (like BaseIntegrationTest); in this case, it has to be marked public